x-pack/plugins/apm/server/lib/correlations/get_correlations_for_failed_transactions/index.ts

```diff
@@ -121,7 +121,7 @@ export async function getErrorRateTimeSeries({
   topSigTerms: TopSigTerm[];
 }) {
   const { start, end, apmEventClient } = setup;
-  const { intervalString } = getBucketSize({ start, end, numBuckets: 30 });
+  const { intervalString } = getBucketSize({ start, end, numBuckets: 15 });
```

Comment:
I think this will still return a dynamic number of buckets to aim for a "nice" bucket size

Reply:
We'll fix this in a separate issue: https://github.com/elastic/kibana/issues/91574